src/main/webapp/script.js

```diff
@@ -46,40 +49,27 @@ function cleanseUrl(url) {
  * Fetches data and adds to html
  */
 function getChart() {
+  $('form').submit(function() {
     const data = new google.visualization.DataTable();
     data.addColumn('string', 'Range');
     data.addColumn('number', 'Count');
-    // TODO: change hardcoded data to sentiment scores (once that part is finished)
-    data.addRows([
-        ['-1.0', null],
-        [null, 1],
-        ['-0.8', null],
-        [null, 1],
-        ['-0.6', null],
-        [null, 4],
-        ['-0.4', null],
-        [null, 5],
-        ['-0.2', null],
-        [null, 10],
-        ['0.0', null],
-        [null, 10],
-        ['0.2', null],
-        [null, 10],
-        ['0.4', null],
-        [null, 10],
-        ['0.6', null],
-        [null, 15],
-        ['0.8', null],
-        [null, 15],
-        ['1.0', null]
-    ]);
+
+    for (currentLabel = LOWEST_SCORE; currentLabel < HIGHEST_SCORE; currentLabel += INTERVAL) {
+      // TODO: Replace abritrary value 6 with correct aggregation value  
+      data.addRows([
+          [(Math.round(currentLabel * 10) / 10).toString(), null],
```

Comment:
Do you think it makes sense to have these values displayed as ranges instead of as lower/upper bounds? Right now the chart is a little bit confusing.

I.e. having the rows having values like "0.2 - 0.4"

Reply:
That's what I had originally, but it was a bit hard to read. The reason I have label, null and then null, value is to to show that it's inbetween the 2 values. Let me know if you want me to try and change it back

Reply:
This makes sense, I agree it should be a range for the labels.